packages/synapse-react-client/src/components/AccessRequirementList/ManagedACTAccessRequirementRequestFlow/DataAccessRequestAccessorsFilesForm.tsx

```diff
@@ -401,14 +431,17 @@ export default function DataAccessRequestAccessorsFilesForm(
                     Upload the completed certificate using the button below:
                   </li>
                 </Typography>
-                <UploadDocumentField
-                  id={'duc'}
-                  uploadCallback={resp =>
-                    uploadCallback(resp, 'ducFileHandleId')
-                  }
-                  documentName={'Data Use Certificate'}
-                  fileHandleAssociations={ducFileHandleAssociation}
-                />
+                <SynapseErrorBoundary>
```

Comment:
Since I removed the try/catch, add an error boundary here--I'd rather fail loudly than silently